src/main/java/seedu/address/ui/WeddingListPanel.java

```diff
@@ -1,26 +1,45 @@
 package seedu.address.ui;
 
+import javafx.beans.binding.Bindings;
+import javafx.beans.binding.StringBinding;
+import javafx.beans.property.ObjectProperty;
 import javafx.collections.ObservableList;
 import javafx.fxml.FXML;
 import javafx.scene.control.ListCell;
 import javafx.scene.control.ListView;
 import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
 import seedu.address.model.wedding.Wedding;
+import seedu.address.model.wedding.WeddingName;
 
 /**
  * Panel containing the list of weddings.
  */
 public class WeddingListPanel extends UiPart<Region> {
     private static final String FXML = "WeddingListPanel.fxml";
 
+    @FXML
+    private VBox currentWeddingNameCardContainer;
+
     @FXML
     private ListView<Wedding> weddingListView;
 
     /**
      * Creates a {@code WeddingListPanel} with the given {@code ObservableList}.
      */
-    public WeddingListPanel(ObservableList<Wedding> weddingList) {
+    public WeddingListPanel(ObservableList<Wedding> weddingList, ObjectProperty<WeddingName> currentWeddingName) {
         super(FXML);
+
+        StringBinding weddingNameBinding = Bindings.createStringBinding(() -> currentWeddingName.get() == null
+                        ? "Not viewing any wedding"
```

Comment:
I think this might be better off written as "Viewing all contacts"

Reply:
I'm not sure if it should be viewing all contacts tho cos it might be a filtered by tags view. Maybe I can add another state for filtered tags "viewing filtered contacts" but then what happens if the person uses filter command in the wedding list, will it still be "viewing wedding ...."